fuel/schemes.py

```diff
@@ -229,6 +229,57 @@ def get_request_iterator(self):
             return imap(list, partition_all(self.batch_size, indices))
 
 
+class BalancedSamplingScheme(ShuffledScheme):
+    """Balanced sampling batches iterator.
+
+    Samples an equal amount of examples from each group and iterates over them
```

Comment:
Should 'group' be class? If this tranformer is intended to be more general than just class up- or downsampling, we should explain this in more detail.